tests/lax_scipy_test.py

```diff
@@ -610,6 +610,12 @@ def test_spectral_dac_svd(self, linear_size, seed, dtype):
     v_eye = jnp.eye(v_unitary_delta.shape[0], dtype=dtype)
     self.assertAllClose(v_unitary_delta, v_eye, atol=eps)
 
+  def test_vq(self):
+    code_book = np.array([[1.,1.,1.], [2.,2.,2.]], dtype=np.float32)
+    features = np.array([[1.9,2.3,1.7], [1.5,2.5,2.2], [0.8,0.6,1.7]], dtype=np.float32)
+    _, dist_min = jsp.cluster.vq.vq(features, code_book)
+    self.assertAllClose(dist_min, np.array([0.43588989, 0.73484692, 0.83066239]))
```

Comment:
Let's use `_CheckAgainstNumpy` to make sure this matches the output of the scipy API, and `_CompileAndCheck` to make sure this works properly under JIT

Reply:
One question: Do `atol` and `rtol` work?
I only see `x == y`
https://github.com/google/jax/blob/e9588ab43fe02e88a7cf92545629eab41ee53f31/jax/_src/test_util.py#L754-L782

Reply:
`atol` and `rtol` work if the arguments are arrays; if they're not arrays it falls back to `x == y`.

Reply:
Ah, I didn't see that the `hasattr(x, '__array__') or np.isscalar(x)` case use `assertArraysAllClose` instead of `assertAllClose`.

Reply:
Done.